### DISCUSSION TITLE
Call for Steering Committee Nominations (Term 1)

### DISCUSSION THREAD

**Discussion by sdc50** [tethysplatform/tethys#916] [Steering Committee Election]
My 3-year term on the Tethys Steering Committee has come to and end, which means it is now time for another election. Prior to the election we will hold a 7-day nomination period (in accordance with the [project by-laws](https://www.tethysplatform.org/project-steering-committee)). Anyone one that is interested in serving on the Tethys Steering Committee may nominate themselves. You may also nominate others that you feel would be a good in this position, but the nominee must express their willingness to serve before they will be eligible to be on the ballot. 

Please express your nominations by replying to this post. After the 7-day period (Tuesday Feb. 7) we will compile all the nominees onto a ballot and send out voting instructions.

Let the nominations begin!

**Comment by swainn** [comment]
I nominate @sdc50 (Scott Christensen)

**Reply by danames** [reply]
I second the nomination if he is willing to continue.

**Reply by sdc50** [reply]
I accept

**Comment by swainn** [comment]
We are 10 days after the time period to submit nominees. Since there is only one nominee (@sdc50) and he has accepted the nomination, we will forgo a formal election.
Congratulations on your re-election to the Tethys Platform PSC and thank you for your continued service.
